config/test.exs

```diff
@@ -1,15 +1,41 @@
 import Config
 
 config :prima_auth0_ex,
+  clients: [],
   authorization_service: AuthorizationServiceMock,
   jwks_kids_fetcher: JwksKidsFetcherMock,
   refresh_strategy: RefreshStrategyMock,
   token_cache: TokenCacheMock,
   token_service: TokenServiceMock
 
 config :prima_auth0_ex, :client,
-  cache_enabled: true,
   token_check_interval: :timer.seconds(1),
   signature_check_interval: :timer.seconds(1)
 
+config :prima_auth0_ex, :test_client,
```

Comment:
Maybe we could use :client configuration options as defaults, that can be overwritten by the individual `:clients` configuration? That seems messy though

Reply:
Just to be sure I understand:

- new clients can be configured with `config :prima_auth0_ex, :clients, client_name: opts`
- for backwards-compatibility we allow `config :prima_auth0_ex, :client`
- we'll throw an error if both `:client` and `:clients` is configured

?

I think that's better, having the `clients` list made explicit was to make it easier to get the clients list and so on, but I think it can still be done even if not defined explicitly.

Reply:
> we'll throw an error if both :client and :clients is configured

I would specifically throw an error if both `:client` and a `:clients, :default_client` were configured together, but other than that yeah, that sounds reasonable